plugin/trino-hive/src/main/java/io/trino/plugin/hive/s3/TrinoS3FileSystem.java

```diff
@@ -295,6 +295,7 @@ public void initialize(URI uri, Configuration conf)
     }
 
     @Override
+    @SuppressWarnings("UnnecessaryMethodReference")
```

Comment:
what is the ignored problem and why you considered it the right thing to do?

at least add a comment, but this can indicate  a bigger problem

Reply:
you can't pass super as a method reference (even if it satisfies functional interface here)

Reply:
like here? 
https://github.com/trinodb/trino/blob/57da5d653d3dd91afba0e057ac6e703752f09078/plugin/trino-phoenix/src/test/java/io/trino/plugin/phoenix/TestPhoenixDistributedQueries.java#L133

or you mean something else?

Reply:
https://github.com/google/error-prone/issues/2102

Reply:
2.5.1 was released with fixes for those false-positives so I'm updating this PR to drop suppressions